galaxy_api/api/ui/serializers/namespace.py

```diff
@@ -34,6 +36,32 @@ class Meta:
             'resources'
         )
 
+    def validate_name(self, field):
+        # NOTE(cutwater): What is that?
+        valid = not not field.strip()
+        if not valid:
+            raise serializers.ValidateError('No name provided')
+        return field
+
+
+    def to_internal_value(self, data):
+        groups = data.get('groups')
+        if groups:
+            data['groups'] = self._sanitize_accounts(groups)
+        return super().to_internal_value(data)
+
+    def _sanitize_accounts(self, accounts):
+        sanitized_groups = []
```

Comment:
I'd suggest this change:

```diff
- sanitized_groups = []
+ sanitized_groups = [auth_models.RH_PARTNER_ENGINEER_GROUP]
```

The `system:partner-engineer` should be included in the groups by default.
The user should not be able to remove it from UI, but sending a request excluding `system:partner-engineer` would still remove it from the namespace.